modules/persistent-storage-local-tolerations.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-local.adoc
+
+[id="local-tolerations_{context}"]
+= Using tolerations with Local Storage Operator Pods
+
+Taints can be applied to nodes to prevent them from running general workloads. To allow the Local Storage Operator to use tainted nodes, you must add tolerations to the Pod or DaemonSet definition. This allows the created resources to run on these tainted nodes.
+
+You apply tolerations to the Local Storage Operator Pod through the LocalVolume resource
+and apply taints to a node through the node specification. A taint on a node is a `key:value pair` that
```

Comment:
From `oc adm taint --help`,
A taint consists of a key, value, and effect. As an argument here, it is expressed as key=value:effect.

Reply:
From https://docs.openshift.com/container-platform/4.2/nodes/scheduling/nodes-scheduler-taints-tolerations.html:
An operator ('Equal' in the following example) allows you to leave one of these parameters empty.

I added this clarification as a note just before the prereqs.

Reply:
User will met https://bugzilla.redhat.com/show_bug.cgi?id=1781520

Reply:
> User will met https://bugzilla.redhat.com/show_bug.cgi?id=1781520

@liangxia I see that this ^^ bug was closed ("will be fixed in master"). So, does this documentation as written look good from QE perspective?

Reply:
Fix in master(currently for 4.4) means it will not fix in 4.3

Reply:
@liangxia I updated key:value in `tolerations` spec to match, as explained in <2> annotation. So, will it work in QE if it uses the `Equal` operator (as described in the note, "An operator allows you to leave one of these parameters empty")?

Based on BZ1781520, I don't understand if all three (key-value-effect) are required, even with Equal/Exists operator. If so, what do you suggest for doc changes?

Reply:
*  A taint consists of a key, value, and effect. As an argument here, it is expressed as key=value:effect.
  *  The key must begin with a letter or number, and may contain letters, numbers, hyphens, dots, and underscores, up to
253 characters.
  *  Optionally, the key can begin with a DNS subdomain prefix and a single '/', like example.com/my-app
  *  The value is optional. If given, it must begin with a letter or number, and may contain letters, numbers, hyphens,
dots, and underscores, up to  63 characters.
  *  The effect must be NoSchedule, PreferNoSchedule or NoExecute.
  *  Currently taint can only apply to node.

I would like we state here that the taint is "key=value:effect", so customer does not forgot adding the effect. Or else, customer will met the confusing error message describe in above bug.

Reply:
And QE confirmed that `Equal` operator does work as stated below in <2> annotation.

Reply:
Thanks, Liang. This makes more sense now. I made a couple of small tweaks to the wording so that it states `key=value:effect`. PTAL and confirm that it looks good.